etherhoused-log

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec papertrail --min-time "24 hours ago" "program:app/web.1 AND (\"turn_off ${1}\" OR \"turn_on ${1}\")"
```

Comment:
Don't we want to retrieve the heroku production logs in order to implement the /log URL? If not, where do all the logs that the daemon writes get sent? How should the log be retrieved on the production system?

Perhaps we should treat etherhoused-log like secrets.sh is treated; we could provide the following in git:
etherhoused-log.example.papertrail
etherhoused-log.example.syslog
...

... and expect the person who installs the package to copy the appropriate example to filename etherhoused-log?